protocols/1479-chinese-university-of-hong-kong/dna_concentration_normalization.ot2.py

```diff
@@ -107,8 +111,14 @@ def csv_to_list(csv_string):
         update_p10_tip_count(1)
 
     # transfer and mix samples
-    total_samples = [well for tuberack in samples for well in tuberack.wells()]
-    for vol, source, dest in zip(dna_vols, total_samples, pcr_plate.wells()):
+    pcr_wells = [well for well in pcr_plate.wells()]
+    tube_wells = [well for rack in tuberack for well in tuberack.wells()]
+    master_list = zip(dna_vols, pcr_wells, tube_wells)
+    starting_well_index = pcr_plate.get_index_from_name('A1')
```

Comment:
Rather than using `A1`, it should be `starting_well` bc its should be a custom input